src/org/scada_lts/utils/QueryUtils.java

```diff
@@ -12,4 +14,5 @@ public static String getArgsIn(int size) {
         args.delete(args.length() - 1, args.length());
         return args.toString();
     }
+
```

Comment:
This should also be thrown out, if there was no change in the class code it should not be shown in the list of changed classes, it obscures the picture.